src/components/body/Body.js

```diff
@@ -1,28 +1,74 @@
 import React, { Component } from 'react';
 import PresentationCard from './PresentationCard/PresentationCard';
+import PersonalHistory from '../../actions/PersonHistory';
+import Ability from '../../actions/Ability';
+
 import Portafolio from './Portafolio/Portafolio';
 import TimeLine from './TimeLine/TimeLine';
 
 class Body extends Component {
+
+  constructor(cVInformation){
```

Comment:
Why this empty constructor?, why `super` without props? , this should be also a stateless component